### DISCUSSION TITLE
Making http requests from content editor's "button links"

### DISCUSSION THREAD

**Discussion by luciandavidescu** [directus/directus#17285] [Feature Requests]
"Button links" are quite handy in that they can be used as contextual triggers for webhooks of sorts.

For example, if there's a field that gets autopopulated manually, it's much more intuitive to have the trigger button right next to it instead of somewhere in a sliding menu; or they can be used to validate that certain fields are filled before proceeding to the next stage and so on.

The big downside is with the ux, as the link is opened in a new window/tab. It would be great if there was this checkable option to fetch the link with an actual http request (possibly with `sendBeacon()`?) instead of opening it.

**Comment by rijkvanzanten** [comment]
Love that idea! It'd be really nice to have those button links be able to execute flows and then update the staged values on the fly 🤔

**Reply by luciandavidescu** [reply]
A few more nice possible things – being able to change other fields + changing own states (clickability, looks etc) depending on certain conditions... that would allow for a proper "publish" button, for example.

**Comment by arladmin** [comment]
Is this something that's already on the roadmap/implemented in some way?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
